features/page_objects/withdraw_registration_page.rb

```diff
@@ -0,0 +1,11 @@
+class WithdrawRegistrationPage < SitePrism::Page
+
+  element(:withdraw_comment, "#admin_enrollment_exemptions_withdraw_comment")
+  element(:submit_button, "input[name='commit']")
+
+  def submit
```

Comment:
Can we stick with the `submit(args = {})` pattern, and pass in the text to use from the step. The reason is I can then look at the step for full details of **what** I'm doing with the page, and the page_object just handles the **how**. To me

```ruby
@app.withdraw_registration_page.submit(comment: "This is a test")
```

is more meaningful in the step in context with everything else I'm doing than simply

```ruby
@app.withdraw_registration_page.submit
```